spec/requests/applicants/omniauth_callbacks_spec.rb

```diff
@@ -47,7 +47,7 @@
 
     it 'does not add its url to page history' do
```

Comment:
Given the expectation of this test has changed, it might be clearer if this was `it 'has no page history' do`.